SPECS/libcriterion/libcriterion.spec

```diff
@@ -0,0 +1,56 @@
+%define pr_name criterion
+Summary:        A cross-platform C and C++ unit testing framework for the 21st century
+Name:           lib%{pr_name}
```

Comment:
Is there a compatibility reason why we don't just use the name `criterion` for the whole package? It seems most packagers besides Ubuntu use `criterion` (and Ubuntu just has a convention to prefix all library package names with `lib`, so we probably don't care much to defer to them). Naming it `criterion` would make it easier for users to find the package.

Reply:
Can we at least provide the `criterion` package name, then, given that that's the name of the project?